Makefile

```
@@ -12,6 +12,22 @@ CONTAINER_IMAGE   = $(IMAGE_REGISTRY)/k8s-website-hugo:v$(HUGO_VERSION)-$(IMAGE_
 # Mount read-only to allow use with tools like Podman in SELinux mode
 # Container targets don't need to write into /src
 CONTAINER_RUN     = "$(CONTAINER_ENGINE)" run --rm --interactive --tty --volume "$(CURDIR):/src:ro,Z"
+CONTAINER_RUN_TTY = "$(CONTAINER_ENGINE)" run --rm --interactive --tty
+CONTAINER_HUGO_MOUNTS = \
+	--read-only \
+	--mount type=bind,source=$(CURDIR)/.git,target=/src/.git,readonly \
+	--mount type=bind,source=$(CURDIR)/archetypes,target=/src/archetypes,readonly \
+	--mount type=bind,source=$(CURDIR)/assets,target=/src/assets,readonly \
+	--mount type=bind,source=$(CURDIR)/content,target=/src/content,readonly \
+	--mount type=bind,source=$(CURDIR)/data,target=/src/data,readonly \
+	--mount type=bind,source=$(CURDIR)/i18n,target=/src/i18n,readonly \
+	--mount type=bind,source=$(CURDIR)/layouts,target=/src/layouts,readonly \
+	--mount type=bind,source=$(CURDIR)/public,target=/src/public \
+	--mount type=bind,source=$(CURDIR)/resources,target=/src/resources \
+	--mount type=bind,source=$(CURDIR)/static,target=/src/static,readonly \
+	--mount type=bind,source=$(CURDIR)/themes,target=/src/themes,readonly \
```

This works for me, locally:
```suggestion
	--mount type=bind,source=$(CURDIR)/.git,target=/src/.git,readonly \
	--mount type=bind,source=$(CURDIR)/archetypes,target=/src/archetypes,readonly \
	--mount type=bind,source=$(CURDIR)/assets,target=/src/assets,readonly \
	--mount type=bind,source=$(CURDIR)/content,target=/src/content,readonly \
        --mount type=bind,source=$(CURDIR)/data,target=/src/data,readonly \
	--mount type=bind,source=$(CURDIR)/i18n,target=/src/i18n,readonly \
	--mount type=bind,source=$(CURDIR)/layouts,target=/src/layouts,readonly \
	--mount type=tmpfs,destination=/src/public,tmpfs-mode=01777 \
	--mount type=tmpfs,destination=/src/resources,tmpfs-mode=01777 \
	--mount type=bind,source=$(CURDIR)/static,target=/src/static,readonly \
	--mount type=bind,source=$(CURDIR)/hugo.toml,target=/src/hugo.toml,readonly \
	--mount type=tmpfs,destination=/tmp,tmpfs-mode=01777
```

---

I made the switch to declaring tmpfs for public and resources (and pushed the commit so you can test too), but I get an error locally:

```console
$ make container-build
mkdir -p public
"docker" run --rm --interactive --tty --read-only --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/.git,target=/src/.git,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/archetypes,target=/src/archetypes,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/assets,target=/src/assets,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/content,target=/src/content,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/data,target=/src/data,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/i18n,target=/src/i18n,readonly --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/layouts,target=/src/layouts,readonly --mount type=tmpfs,destination=/src/public,tmpfs-mode=01777 --mount type=tmpfs,destination=/src/resources,tmpfs-mode=01777 --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/static,target=/src/static,readonly --mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 --mount type=bind,source=/Users/chalin/git/lf/kubernetes/kubernetes.io/hugo.toml,target=/src/hugo.toml,readonly gcr.io/k8s-staging-sig-docs/k8s-website-hugo:v0.133.0-7886d9227c22 \
                hugo --cleanDestinationDir --buildDrafts --buildFuture --environment preview --noBuildLock
Start building sites … 
hugo v0.133.0+extended linux/amd64 BuildDate=unknown

Total in 1568 ms
Error: error copying static files: chtimes /src/public: operation not permitted
```

I'm investigating further, but wanted you to mention that I'm getting an error.

---

Tthe solution for the containerized build is to use ` --destination /tmp/public`, at least that works for me now. Can you give it a try @sftim, I just pushed the change.